src/orchestratord/src/controller/materialize.rs

```diff
@@ -59,6 +59,8 @@ pub struct Args {
     #[clap(long)]
     enable_security_context: bool,
 
+    #[clap(long)]
+    orchestratord_pod_selector_labels: Vec<KeyValueArg<String, String>>,
```

Comment:
this can also go away since we're no longer using it

Reply:
This is still used, but I did need to still pass it to orchestratord, which is now fixed.